front/src/applications/editor/Editor.tsx

```diff
@@ -18,8 +18,9 @@ import TOOLS from 'applications/editor/tools/constsTools';
 import { useSwitchTypes } from 'applications/editor/tools/switchEdition/types';
 import type { switchProps } from 'applications/editor/tools/switchProps';
 import type { CommonToolState } from 'applications/editor/tools/types';
+import useInfra from 'applications/editor/tools/useInfra';
 import { centerMapOnObject, selectEntities } from 'applications/editor/tools/utils';
-import { osrdEditoastApi, type ObjectType } from 'common/api/osrdEditoastApi';
+import type { ObjectType } from 'common/api/generatedEditoastApi';
```

Comment:
Please always use `common/api/osrdEditoastApi` unconditionally! `generatedEditoastApi` should never be imported directly.